docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -160,63 +135,49 @@ The placement policy above has 4 followers:
 
 Behavior notes:
 
-* Advanced placement is available in the context of `CREATE|ALTER PLACEMENT POLICY`, `CREATE|ALTER DATABASE` and `CREATE|ALTER TABLE`. i.e. the usage of all placement syntax is expected to be the same in all contexts.
+* Advanced placement is available in the context of `CREATE|ALTER PLACEMENT POLICY`, i.e. the usage of all placement syntax is expected to be the same in all contexts.
 * It is possible to set `CONSTRAINTS`, `LEADER_CONSTRAINTS`, `FOLLOWER_CONSTRAINTS` and `LEARNER_CONSTRAINTS`. Assuming that both `CONSTRAINTS` and `FOLLOWER_CONSTRAINTS` are specified, the conditions are "AND"ed together.
 * See "Constraints configuration" below for a full set of rules and syntax for constraints.
 
 #### Metadata commands
 
-Besides `SHOW CREATE PLACEMENT POLICY x` and `SHOW CREATE TABLE t1` it should be possible to summarize all placement for a database system. This is beneficial for compliance scenarios.
+Besides `SHOW CREATE PLACEMENT POLICY x`, it should be possible to summarize all placement for a database system. This is beneficial for compliance scenarios.
 
-##### information_schema.placement_rules
+##### information_schema.placement_policies
```

Comment:
So there is a renaming. And just to be sure, now it only views policies and not tables/partitions? If so, we need an update.

Reply:
That's right; it only shows partitions now. This is https://github.com/pingcap/tidb/issues/31425 to update.

The reason for the rename: `placement_rules = (direct + placement policy)`. Since the `infoschema.partitions` and `infoschema.tables` tables have a column for tidb_placement_policy we don't need it in this table.